Patches/HudPatch.cs

```diff
@@ -119,7 +119,7 @@ public static void Postfix(HudManager __instance)
                     __instance.KillButton.ToggleVisible(false);
                 }
 
-                bool CanUseVent = player.CanUseImpostorVentButton();
+                bool CanUseVent = player.CanUseVent();
                 __instance.ImpostorVentButton.ToggleVisible(CanUseVent);
```

Comment:
Found bug, Impostor vent button will be visible for engineer based (for modded)

Reply:
it should not be changed here i think